packages/core/analyzer-import-alias-plugin/src/index.ts

```diff
@@ -1,25 +1,40 @@
+import { Plugin } from '@custom-elements-manifest/analyzer';
```

Comment:
This isn't actually the plugin code we need - that will be addressed in a subsequent PR. This PR is just implementing the minimal required change to convert this existing file to typescript (that is why there is such liberal use of `any`).